File: Java1/Array.java
class Array{
  public static void main(String[] args){
    int[] score = new int[3]; //配列の生成
    score[0] = 80;
    score[1] = 100;
    score[2] = 75;

    String[] name = {"宗像", "菅原", "田中"}; //配列の初期化

    System.out.println(name[0] + "さん：" + score[0] + "点");
    System.out.println(name[1] + "さん：" + score[1] + "点");
    System.out.println(name[2] + "さん：" + score[2] + "点");
    System.out.println("受験者数：" + score.length + "名");
  }
}

File: Java1/Array.rb
# 点数
score = [80, 100, 75]
# 名前
name = ["鈴木", "田中", "宗像"]

puts name[0] + "さん" + "#{score[0]}" + "点"
puts name[1] + "さん" + "#{score[1]}" + "点"
puts name[2] + "さん" + "#{score[2]}" + "点"
puts "受験者数" + score.length.to_s + "名"
puts "得点合計：" + "#{score[0] + score[1] + score[2]}"